configure.ac

```diff
@@ -540,15 +540,12 @@ AS_IF([test "$enable_DDR" = yes -a "$cross_compiling" != "yes"],
 					[
 					# this should yield .../VC/bin/cl.exe or .../VC/bin/amd64/cl.exe
 					vc_bin_dir=$(AS_DIRNAME(["$VS_CL_PATH"]))
-					diasdk_dir="$vc_bin_dir/../../DIA SDK"
-					AS_IF([test -f "$diasdk_dir/include/dia2.h"]
-						[DIASDK_HOME="$(cygpath -a -w "$diasdk_dir")"],
-						[
-						diasdk_dir="$vc_bin_dir/../../../DIA SDK"
-						AS_IF([test -f "$diasdk_dir/include/dia2.h"],
-							[DIASDK_HOME="$(cygpath -a -w "$diasdk_dir")"]
-						)
-						]
+					diasdk_dir1="$vc_bin_dir/../../DIA SDK"
+					diasdk_dir2="$vc_bin_dir/../../../DIA SDK"
+					diasdk_dir3="$vc_bin_dir/../../../../../../../DIA SDK"
```

Comment:
For `diasdk_dir3`, please provide example absolute paths that are covered by this addition.

Reply:
diasdk_dir3 is used to cover the case of VS2017 (works for both Professional and Community version). The reason is that the path to cl.exe changes on VS2017.
```
Administrator@JCHTORWIN101 /cygdrive/c/Program Files (x86)/Microsoft Visual Studio/2017/Professional
$ ls
 Common7      'DIA SDK'         JS         SDK           VB    VSSDK
 CoreCon       ImportProjects   Licenses  'Team Tools'   VC    Web
 DesignTools   JavaScript       MSBuild    TS            VC#   Xml

Administrator@JCHTORWIN101 /cygdrive/c/Program Files (x86)/Microsoft Visual Studio/2017/Professional
$ find . -name cl.exe
./VC/Tools/MSVC/14.14.26428/bin/Hostx64/x64/cl.exe
./VC/Tools/MSVC/14.14.26428/bin/Hostx64/x86/cl.exe
./VC/Tools/MSVC/14.14.26428/bin/Hostx86/x64/cl.exe
./VC/Tools/MSVC/14.14.26428/bin/Hostx86/x86/cl.exe
```
e.g.
In VS2017 Professional, the path to DIA SDK:  C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\DIA SDK while vc_bin_dir is C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\VC\Tools\MSVC\14.14.26428\bin\Hostx86\x64\cl.exe.  So, we need to step backwards from the path to cl.exe by 7 level to  reach the DIA SDK directory.

Reply:
Thanks for the explanation.